.github/workflows/e2e.yaml

```diff
@@ -32,6 +32,12 @@ jobs:
           file: Dockerfile.base
           AWS_REGION: ${{ secrets.QA_AWS_REGION }}
           AWS_ROLE_TO_ASSUME: ${{ secrets.QA_AWS_ROLE_TO_ASSUME }}
+      - name: Base Image Built
+        run: |
+          echo "### chainlink image used for this test run :link:" >>$GITHUB_STEP_SUMMARY
+          echo "\`${{ env.CHAINLINK_VERSION }}\`" >>$GITHUB_STEP_SUMMARY
+          echo "### test-base-image image tag for this test run :ship:" >>$GITHUB_STEP_SUMMARY
+          echo "\`ci.${{ github.sha }}\`" >>$GITHUB_STEP_SUMMARY
```

Comment:
Can we include the same for chainlink integration-tests workflow as well?

Reply:
Already exists for smoke, soak, and benchmark tests in chainlink!